app/controllers/spree/admin/resource_controller_decorator.rb

```diff
@@ -0,0 +1,8 @@
+Spree::Admin::ResourceController.class_eval do
+  def parent
+    if parent_data.present?
+      @parent ||= parent_data[:model_class].find_by("#{parent_data[:find_by]}": params["#{model_name}_id"])
+      instance_variable_set("@#{model_name}", @parent)
+    end
```

Comment:
@tvdeyen, this in regards to #26. 
When we update solidus to be more flexible with find_by - we will remove this.